### PR TITLE
"Standardize" publication authors list

### PR DESCRIPTION
**Pull request by manzt**
Tries to make the cite authors consistent with Nils' preference of "Initial(s) Last, Intial(s) Last, etc"

### CODE REVIEW

File: _publications/keller-2024-use-coordination-osf.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ type: preprint
...
 
-The source code and documentation are available under the MIT License at [https://github.com/keller-mark/use-coordination](https://github.com/keller-mark/use-coordination).
\ No newline at end of file
+The source code and documentation are available under the MIT License at [https://github.com/keller-mark/use-coordination](https://github.com/keller-mark/use-coordination).
```
Comment (dvdkouril) [inline _publications/keller-2024-use-coordination-osf.md:20]: no change

File: _publications/lyi-2023-cistrome-explorer.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ type: article
 
 publisher: "https://doi.org/10.1093/bioinformatics/btad018"
 cite:
-  authors: "Sehi L'Yi, Mark S Keller, Ariaki Dandawate, Len Taing, Chen-Hao Chen, Myles Brown, Clifford A Meyer, Nils Gehlenborg"
+  authors: "S L'Yi, MS Keller, A Dandawate, L Taing, CH Chen, M Brown, CA Meyer, N Gehlenborg"
```
Comment (dvdkouril) [inline _publications/lyi-2023-cistrome-explorer.md:15]: Is there a standard for these hyphenated names? `Chen-Hao Chen` -> `C-H Chen` or `CH Chen`?

File: _publications/lyi-2024-cistrome-data-browser.md [hunk 3]
```diff
@@ -11,8 +11,8 @@ type: article
...
...me Data Browser is a resource of ChIP-seq, ATAC-seq and DNase-seq data from humans and mice. It provides maps of the genome-wide locations of transcription factors, cofactors, chromatin remodelers, histone post-translational modifications and regions of chromatin accessible to endonuclease activity. Cistrome DB v3.0 contains approximately 45,000 human and 44,000 mouse samples with about 32,000 newly collected datasets compared to the previous release. The Cistrome DB v3.0 user interface is implemented as a single page application that unifies menu driven and data driven search functions and provides an embedded genome browser, which allows users to find and visualize data more effectively. Users can find informative chromatin profiles through keyword, menu, and data-driven search tools. Browser search functions can predict the regulators of query genes as well as the cell type and factor dependent functionality of potential cis-regulatory elements. Cistrome DB v3.0 expands the display of quality control statistics, incorporates sequence logos into motif enrichment displays and includes more expansive sample metadata. Cistrome DB v3.0 is available at http://db3.cistrome.org/browser.
```
Comment (dvdkouril) [inline _publications/lyi-2024-cistrome-data-browser.md:18]: No change

**Review by dvdkouril — COMMENTED** [review]
Did a readthrough, some comments.

File: _publications/gehlenborg-2013-cell-glioblastoma.md [hunk 4]
```diff
@@ -7,7 +7,7 @@ type: article
...
+  authors: "CW Brennan, RG Verhaak, A McKenna, B Campos, H Noushmehr, SR Salama, S Zheng, D Chakravarty, JZ Sanborn, SH Berman, R Beroukhim, B Bernard, CJ Wu, G Genovese, I Shmulevich, J Barnholtz-Sloan, L Zou, R Vegesna, SA Shukla, G Ciriello, WK Yung, W Zhang, C Sougnez, T Mikkelsen, K Aldape, DD Bigner, EG Van Meir, M Prados, A Sloan, KL Black, J Eschbacher, G Finocchiaro, W Friedman, DW Andrews, A Guha, M Iacocca, BP O’Neill, G Foltz, J Myers, DJ Weisenberger, R Penny, R Kucherlapati, CM Perou, DN Hayes, R Gibbs, M Marra, GB Mills, E Lander, P Spellman, R Wilson, C Sander, J Weinstein, M Meyerson, S Gabriel, PW Laird, D Haussler, G Getz, L Chin (The Cancer Genome Atlas Network, incl. N Gehlenborg)"
```
[the remark this answers is not in the document]
Comment (sehilyi) [reply _publications/gehlenborg-2013-cell-glioblastoma.md:10]: I think the version that @dvdkouril suggested is the accurate one. The consortium is considered another "author" in such papers.

File: _publications/gehlenborg-2020-medrxiv.md [hunk 5]
```diff
@@ -12,7 +12,7 @@ type: article
...
+  authors: "GA Brat, GM Weber, N Gehlenborg, P Avillach, NP Palmer, L Chiovato, J Cimino, LR Waitman, GS Omenn, A Malovini, JH Moore, BK Beaulieu-Jones, V Tibollo, SN Murphy, SL' Yi, MS Keller, R Bellazzi, DA Hanauer, A Serret-Larmande, AG Gutierrez-Sacristan, DS Bell, KD Mandl, RW Follett, JG Klann, DA Murad, L Scudeller, M Bucalo, K Kirchoff, J Craig, J Obeid, V Jouhet, R Griffier, S Cossin, B Moal, L Patel, A Bellasi, HU Prokosch, D Kraska, P Sliz, KY Ngiam, A Zambelli, E Schiver, B Devkota, RL Bradford, C Daniel, V Benoit, R Bey, N Paris, P Serre, N Orlova, J Dubiel, M Hilka, AS Jannot, J Leblanc, N Griffon, A Burgun, M Bernaux, A Sandrin, E Salamanca, S Cormont, T Ganslandt, T Gradinger, J Champ, M Boeker, P Martel, L Esteve, A Gramfort, O Grisel, D Leprovost, T Moreau, G Varoquaux, JJ Vie, D Wassermann, A Mensch, C Caucheteux, C Haverkamp, G Lemaitre, S Bosari, ID Krantz, A South, ALM Tan, T Cai, IS Kohane (for the 4CE Consortium)"
```
[the remark this answers is not in the document]
Comment (sehilyi) [reply _publications/gehlenborg-2020-medrxiv.md:15]:
```suggestion
  authors: "GA Brat, GM Weber, N Gehlenborg, P Avillach, NP Palmer, L Chiovato, J Cimino, LR Waitman, GS Omenn, A Malovini, JH Moore, BK Beaulieu-Jones, V Tibollo, SN Murphy, S L'Yi, MS Keller, R Bellazzi, DA Hanauer, A Serret-Larmande, AG Gutierrez-Sacristan, DS Bell, KD Mandl, RW Follett, JG Klann, DA Murad, L Scudeller, M Bucalo, K Kirchoff, J Craig, J Obeid, V Jouhet, R Griffier, S Cossin, B Moal, L Patel, A Bellasi, HU Prokosch, D Kraska, P Sliz, KY Ngiam, A Zambelli, E Schiver, B Devkota, RL Bradford, C Daniel, V Benoit, R Bey, N Paris, P Serre, N Orlova, J Dubiel, M Hilka, AS Jannot, J Leblanc, N Griffon, A Burgun, M Bernaux, A Sandrin, E Salamanca, S Cormont, T Ganslandt, T Gradinger, J Champ, M Boeker, P Martel, L Esteve, A Gramfort, O Grisel, D Leprovost, T Moreau, G Varoquaux, JJ Vie, D Wassermann, A Mensch, C Caucheteux, C Haverkamp, G Lemaitre, S Bosari, ID Krantz, A South, ALM Tan, T Cai, IS Kohane (for the 4CE Consortium)"
```

**Comment by manzt** [comment]
Thanks for the review... wasn't quite ready. I wrote a new script to harmonize the publications here based on their zotero keys. I'm going to open a separate PR.

**Pull request closed without merging**